README.md

```diff
@@ -122,7 +122,7 @@ provider "kubernetes" {
 }
 ```
 
-To try the module, please run `terraform apply` command in `test/fixture` folder.
+To try the module, please run `terraform apply` command in `examples`'s sub folder.
```

Comment:
I would rephrase this way:

Some examples to try the module, are available in the `examples` folder. These examples are tested against every PR with the [E2E test](https://github.com/Azure/terraform-azurerm-aks/blob/master/test/e2e/terraform_aks_test.go).